plugins/chainlink.Dockerfile

```diff
@@ -75,7 +75,8 @@ COPY --from=buildgo /go/bin/dlv /usr/local/bin/dlv
 # Set plugin environment variable configuration.
 ENV CL_MEDIAN_CMD=chainlink-feeds
 ENV CL_MERCURY_CMD=chainlink-mercury
-ENV CL_SOLANA_CMD=chainlink-solana
+ARG CL_SOLANA_CMD
+ENV CL_SOLANA_CMD=${CL_SOLANA_CMD}
```

Comment:
Why do this? This has been stable for _years_, and IIUC will break tests that assume it is enabled.

Reply:
How do we disable it for CCIP?

Reply:
It is an environment variable that can be toggled at runtime.

Reply:
Setting this back in https://github.com/smartcontractkit/chainlink/pull/17591.